Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Sliders/PinchSlider.cs

```diff
@@ -31,7 +31,7 @@ public GameObject ThumbRoot
                 thumbRoot = value;
                 InitializeSliderThumb();
             }
-        }
+        } 
```

Comment:
```suggestion
        }
```

Reply:
Not sure what's the recommended change is?

Reply:
Ah just removing an extra space at the end of the line. Not super important!

Reply:
ok, I made this change as well :)